README.md

```diff
@@ -1,3 +1,56 @@
+# Contributors
```

Comment:
Not sure, whether this should be on the very top of the README. 

@lustoykov What do you think?

Reply:
👍